bin/yoke

```diff
@@ -20,6 +20,11 @@ try:
     service.run()
 except KeyboardInterrupt:
     pass
+except OSError as err:
+    if err.errno == 98:
```

Comment:
This is [`errno.EADDRINUSE`](https://docs.python.org/3/library/errno.html#errno.EADDRINUSE), right? I think we can just `import errno`

Reply:
Okay, perfect. It's much clearer now. 1acd6c5c261dd11554f3a7357401b7c8314582d2